generated/python/grpc-google-cloud-logging-v2/setup.py

```diff
@@ -33,11 +33,11 @@
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: Implementation :: CPython',
   ],
-  description='GRPC library for the google-logging-v2 service',
+  description='GRPC library for the Google Logging service',
```

Comment:
The change is autogenerated; the difference is because I changed the gRPC template to match the GAPIC one here. I just forgot to apply the same handedit.

And yup, that's still the plan to derive this from the service config. I'm planning to attack this (and several of the other hand-edits) through a change to the artman config file, tracked here: https://github.com/googleapis/artman/issues/116